src/InputContainer.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { updateTaskTitle, addTask } from './actions';
+
+import InputContainer from './InputContainer';
+
+describe('InputContainer', () => {
+  const renderInputContainer = () => render(<InputContainer />);
+
+  const dispatch = jest.fn();
+
+  useSelector.mockImplementation((selector) => selector({
+    taskTitle: 'TDD는 언제나 새롭다',
+  }));
+
+  beforeEach(() => jest.clearAllMocks());
+
+  it('renders title', () => {
+    const { getByDisplayValue } = renderInputContainer();
+
+    expect(getByDisplayValue(/TDD는 언제나 새롭다/)).not.toBeNull();
+  });
+
+  it('updates value upon changing of input value', () => {
+    useDispatch.mockImplementation(() => dispatch);
+
+    const { getByDisplayValue } = renderInputContainer();
+
+    fireEvent.change(getByDisplayValue(/TDD는 언제나 새롭다/), { target: { value: '거짓말 아님' } });
```

Comment:
두개의 인자를 받는데 두번째 인자는 객체 형태라서 이 경우에는 가독성을 위해 줄바꿈 처리해주는건 어떨까요?
```suggestion
    fireEvent.change(getByDisplayValue(/TDD는 언제나 새롭다/), {
      target: {
        value: '거짓말 아님' 
      }
    });
```